### ISSUE TITLE
The search speed of bananas algorithm compare with random is quite solow?

### ISSUE THREAD

**Issue by auroua** [naszilla/bananas#8]
The paper said  ```We note that a plot with respect to wall-clock time would look
nearly identical, since all NAS algorithms finish the set of 150 queries in roughly the same amount
of time. The fastest, random search, takes 46.5 TPU hours, while BANANAS, the slowest, takes
47.0 TPU hours```.
I am running the code `run_experiments_sequential.py` in this repository. The `walltimes` of 150 queries with algorithms `bananas`, `random`, `evolution` is quite different. The `bananas` algorithms is ten times slower than random. The following line is the output:
`[359.68584871292114, 35.89821910858154, 35.73087739944458]` 
Does I made any mistakes?

**Comment by crwhite14** [comment]
Hi, it sounds like you are running the algorithms for the NASBench dataset. Please look at the next sentence after the one you quoted above: `the average time to train a NASBench architecture is 18.6 minutes on a TPU`.

Recall that the NASBench dataset has precomputed validation error for all the architectures, designed to make research in NAS quick/easy. To compute the *real* runtime, we must add 150 * (18.6 TPU minutes). Then we will get 46.5 and 47.0 TPU hours. Please let us know if that helps, or if you have more questions.

**Comment by auroua** [comment]
I think it's unfair as lots of time is occupied by training the sampled architecture.

**Comment by crwhite14** [comment]
Hi, please remember that for real NAS experiments, the bottleneck by far is training each chosen architecture. For example on DARTS, it takes 2-3 GPU hours to train a new architecture, and only a few minutes to train the meta neural network ensemble. On NASBench, it takes 18.6 minutes to train an architecture, and less than 2 minutes to train the meta neural network. Also remember that in each iteration, we train 10 new architectures and train the meta neural network ensemble once.

Random search: 150 * (18.6 TPU minutes) + (35 CPU seconds) = 46.5 TPU hours
BANANAS: 150 * (18.6 TPU minutes) + 15 * (2 CPU minutes) = 47.0 TPU hours

This is the same way that runtimes are calculated in the original NASBench paper. Google released the precomputed validation and test accuracy for every architecture in NASBench so that research will be quick and easy. But it would be meaningless to report the runtime when the architecture accuracies are precomputed. Please let us know if you have more questions.

**Comment by auroua** [comment]
Thanks for your replay!
